frontend/js/index.js

```diff
@@ -89,19 +97,23 @@ const compareButton = () => {
             <td class="td-index" title="${filename}">${filename}</td>
             <td class="td-index">${machine}</td>
             <td class="td-index" style="width: 120px"><span title="${last_run}">${dateToYMD(new Date(last_run))}</span></td>
-            <td><input type="checkbox" value="${id}" name="chbx-proj"/>&nbsp;</td>`;
+            <td><input type="checkbox" value="${id}" name="chbx-proj" id="chbx-proj" />&nbsp;</td>`;
```

Comment:
Thanks for the great work. I just scanned the PR and noticed that `name` and `id` are note unique. Somehow `name` not being unique [1] must have slipped through our review process. (Probably my fault :) Probably best to solve this with a `class` and then select on `.js-somename`. 

We can either merge this now and then refactor right away (@djesic) or do it in this PR. The latter being my preferred way if you could do this?
 
[1] https://www.w3schools.com/html/html_id.asp#:~:text=The%20id%20attribute%20specifies%20a,element%20with%20the%20specific%20id.
```
The id attribute specifies a unique id for an HTML element. The value of the id attribute must be unique within the HTML document.
```

Reply:
Name is not unique by design. When you have mutliple checkboxes with the same name they will be understood as array. In GET as well as in POST.

Reply:
Totally right. I got confused when looking at the `id` documentation.